docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a Quick Fix (automatic code replacement) or by editing the code.
```

Comment:
Try to avoid parentheses. Many writing teams do not allow them. Rewrite this to define the term, Quick Fix, so that you do not need parentheses.